qa-core/src/config/internal-api/TestConfiguration/tables.ts

```diff
@@ -0,0 +1,64 @@
+import { Response } from "node-fetch"
+import { Row, Table } from "@budibase/types"
+import InternalAPIClient from "./InternalAPIClient"
+import { responseMessage } from "../fixtures/types/responseMessage"
+
+
+export default class TablesApi {
+    api: InternalAPIClient
+
+    constructor(apiClient: InternalAPIClient) {
+        this.api = apiClient
+    }
+
+    async getTables(expectedNumber: Number): Promise<[Response, Table[]]> {
+        const response = await this.api.get(`/tables`)
+        const json = await response.json()
+        expect(response).toHaveStatusCode(200)
+        expect(json.length).toBe(expectedNumber)
+        return [response, json]
+    }
+
+    async getTableById(id: string): Promise<[Response, Table]> {
+        const response = await this.api.get(`/tables/${id}`)
+        const json = await response.json()
+        return [response, json]
+    }
+
+    async create(body: any): Promise<[Response, Table]> {
```

Comment:
should probably call this `save`, given it's used for the create and the update

Reply:
Done